indico/modules/categories/controllers/display.py

```diff
@@ -125,13 +125,15 @@ def _category_query_options(cls):
         children_strategy.subqueryload('acl_entries')
         children_strategy.undefer('deep_children_count')
         children_strategy.undefer('deep_events_count')
-        children_strategy.undefer('has_events')
+        children_strategy.undefer('has_events'),
+        children_strategy.undefer('has_children'),
```

Comment:
Those are normal function calls, not a list of arguments ;)

```suggestion
        children_strategy.undefer('has_events')
        children_strategy.undefer('has_children')
```